### ISSUE TITLE
get code

### ISSUE THREAD

**Issue by lina-code-FER** [ShaoXiang23/CLIP_Hand_Demo#4]
Hi author, can you provide the training code, and the definition of the loss function, thanks a lot!
